docs/essentials/files.md

```diff
@@ -0,0 +1,57 @@
+---
+id: essentials_files
+title:  "File Channel"
+---
+
+A `AsynchronousFileChannel` provides API for handling files in non-blocking way.
+
+Required imports for presented snippets:
+
+```scala mdoc:silent
+import java.nio.file.Paths
+import zio._
+import zio.nio.channels._
+import zio.console._
+```
+
+## Basic operations 
+
+Opening file for given path and no additional open attributes:
+
+```scala mdoc:silent
+val channelM = for {
+  path <- ZIO.succeedLazy(Paths.get("./file.txt"))
```

Comment:
`succeedLazy` is about to be deprecated from `ZIO`, therefore I'd argue that we shouldn't include it in examples.

Reply:
Changed to `effectTotal`